client-core/src/service/wallet_service.rs

```diff
@@ -101,11 +103,15 @@ pub struct WalletInfo {
     /// wallet meta data
     #[serde(deserialize_with = "deserde_from_str", serialize_with = "serde_to_str")]
     pub wallet: Wallet,
-    /// private key of the wallet
+    /// private key of view key
     #[serde(deserialize_with = "deserde_from_str", serialize_with = "serde_to_str")]
     pub private_key: PrivateKey,
     /// passphrase used when import wallet
     pub passphrase: Option<SecUtf8>,
+    /// private_key: public_key-> private_key paire
+    pub key_pairs: BTreeMap<PublicKey, Option<Vec<u8>>>,
+    /// multisig_address: hex encoded root_hash -> multisig_address pair
+    pub multisig_addresses: BTreeMap<String, Vec<u8>>,
```

Comment:
`MultiSigAddress` scale-encoded is the Vec<u8>?

Reply:
yes, it's parity_scale_codec encoded multisig_address.